cli/functionary/package.py

```diff
@@ -61,6 +60,22 @@ def create_cmd(ctx, language, name, output_directory):
     shutil.copytree(str(basepath), str(dir), dirs_exist_ok=True)
     generateYaml(output_directory, name, language)
 
+    click.echo()
+    click.echo(f"Package creation for {name} successful!\n")
+    text = Text()
+    console = Console()
+    text.append("Next Steps: \n", style="b u blue")
```

Comment:
I think this looks much better now.  A few super minor suggestions.

* "Next Steps: " I think you can drop the colon.  I'm 50/50 on if the underline is necessary.  The bold and the color choice make it stand out enough to me, but I'll let you decide if the underline stays or goes.
* Drop the trailing space before the newline on each of these. (e.g. `"Next Steps\n"` rather than `"Next Steps \n"`)
* Let's add an additional newline before the publish command line.  I think that helps make it more readable.